src/exe/build_image_assets/build_image_assets.cpp

```diff
@@ -20,7 +20,7 @@ map<string, string> getFileBlobs(string dir_path)
 	map<string, string> blobs;
 
 	vector<string> anchors = {"anchor-{}.png", "anchor-secondary-{}.png", "guide-horizontal-{}.png", "guide-vertical-{}.png"};
-	for (const string& mode : {"light", "dark"})
+	for (const auto& mode : {"light", "dark"})
```

Comment:
Some compiler nonsense I've forgotten. Whatever, this is why God invented `auto`